appengine-standard-java8/springboot-appengine-standard/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -14,4 +14,5 @@ limitations under the License.
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <threadsafe>true</threadsafe>
   <runtime>java8</runtime>
+  <property name="java.util.logging.config.file" value="WEB-INF/classes/logging.properties"/>
```

Comment:
You could just increase the kind of 
```
  <instance-class>B8</instance-class>`
  <manual-scaling>
    <instances>5</instances>
  </manual-scaling>
```

Reply:
Increasing the instance size seems not a great workaround (given I'm not sure you're guaranteed to not run into this out of memory error even then)